.travis.yml

```diff
@@ -29,6 +29,7 @@ env:
     - PA_CLIENT_ID=fakeclientid.apps.googleusercontent.com
```

Comment:
Please add a Test Plan to your PR description; it should outline that you've deployed it to dev, what you've tested while deployed to dev, and that the normal chatbot flow completes.

Take a look at Elana's PR for an example test plan: https://github.com/bravetechnologycoop/BraveButtons/pull/226

Or one of my previous test plans: https://github.com/bravetechnologycoop/BraveButtons/pull/216

Reply:
done